### ISSUE TITLE
Significance of arguments follow_targets and compute_targets in call to eval function in infer.py

### ISSUE THREAD

**Issue by ahanagemini** [mitroadmaps/roadtracer#20]
What is the significance of the arguments follow_targets and compute_targets in call to eval function in infer.py?

**Comment by uakfdotb** [comment]
This is for evaluating loss and accuracy. By default follow_targets=False so that we use the machine learning model to determine the angles to follow.

compute_targets can be True even if follow_targets=False, in which case some additional features can be added to the output visualizations (if SAVE_EXAMPLES=True). But if compute_targets=True, then ground truth data must be supplied.

If follow_targets=True, then the angles will be determined based on the ground truth data. This option is used during training to evaluate the loss/accuracy over validation set. Otherwise comparing the loss/accuracy doesn't make sense because the inputs will be different as the model learns, since follow_targets=False means that the input on one iteration is determined based on the angles chosen in previous iterations.

**Comment by ahanagemini** [comment]
Thank you. That clears things up.
